### PR TITLE
Small revision on record, users, social pages

### PR DESCRIPTION
**Pull request by TakLee96**

### CODE REVIEW

**Comment by rickmak** [comment]
Only things concern me is the `_id` and `id`. I think we should make normal operation free from `_id` and only use `id` in the suggested way of coding in document and example. But this is out of the PR scope. refs https://github.com/SkygearIO/skygear-SDK-JS/issues/22

Other looks good to me. reassigning to @david90 for language and merges.

**Comment by TakLee96** [comment]
@rickmak shall I just assume that `query.equalTo("id", "note/123...");` will work in the future and update the doc accordingly in the next PR?

File: content/js/relation.md [hunk 1]
```diff
@@ -1,76 +1,91 @@
...
 skygear.relation.queryFollower(ben).then((users) => {
   console.log(users);
 }, (error) => {
-  console.warn(error);
+  console.error(error);
 });
 ```
 
-Get follower 
+To get current user's friend list:
```
Comment (david90) [inline content/js/relation.md]: *the* current user

**Comment by david90** [comment]
Seen, other than that should be fine

**Comment by TakLee96** [comment]
Ok, I will add `the` right now.

**Pull request merged**